doc/content/getting-started/console/profile-settings/_index.md

```diff
@@ -0,0 +1,43 @@
+---
+title: "Profile Settings"
+description: ""
+weight: 1
+---
+
+This section explains how to modify profile settings on in {{% tts %}} Console.
+
+<!--more-->
+
+Once you are logged into the Console, you can adjust profile settings. Click on your avatar in the upper right corner and select **Profile settings**.
+
+{{< figure src="profile-settings.png" alt="Navigate to profile settings" >}}
+
+You will be redirected to a dedicated profile settings page.
+
+## General Settings
+
+To change basic info such as your profile picture, name or email address, click the **Expand** button next to the **General settings** section.
+
+For profile picture, the [Gravatar](https://en.gravatar.com/) image associated with your email address is used by default. You can choose to **Upload an image** to change your profile picture.
+
+{{< note >}} You cannot change the **User ID**, but you can set your name to be displayed instead by filling in the **Name** field. {{</ note >}}
+
+You can change the primary email address associated with your {{% tts %}} account by entering a new one in the **Email address** field.
+
+{{< figure src="general-settings.png" alt="General settings" >}}
+
+Click **Save changes** to save any modified info.
+
+## Change password
+
+To set up a new password for your account, click the **Expand** button next to the **Change password** section.
+
+Type in your **Current password**, then create a **New password** and confirm it.
+
+{{< note >}} Your password must be at least 8 characters long, and must contain at least 1 uppercase character and at least 1 digit. We recommend you create a strong password to enhance your profile's security. {{</ note >}}
```

Comment:
Note that this is configurable, so these values are not always the case.

Reply:
how can one configure this?